src/collector/parsers/typescript.ts

```diff
@@ -31,18 +31,30 @@ export interface IClass {
     name: string,
     description: string
     tags: ITag[]
-    properties: any
+    properties: IProperty[]
     methods: IMethod[]
     accessors: any
 }
 
+export interface IProperty {
+    name: string,
+    description: string,
+    returnType: string,
+    visibility: string,
```

Comment:
please remove the commas

Reply:
Fixed on https://github.com/spryker-sdk/frontend-sniffer/pull/3